sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/StreamingQueryStatusListener.scala

```diff
@@ -115,7 +115,7 @@ private[sql] class StreamingQueryStatusListener(
   }
 }
 
-private[sql] class StreamingQueryData(
+private[spark] class StreamingQueryData(
```

Comment:
Why do we need this change?

Reply:
This is because `StreamingQueryData` is not accessible from `SqlResource`

Reply:
Got it. You are right. `org.apache.spark.status.api.v1.sql.SqlResource` is not under `org.apache.spark.sql`.